metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -139,6 +140,10 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     extract_lineage: bool = pydantic.Field(
         default=True, description="Whether lineage should be ingested"
     )
+    # Enable/Disable extracting endorsements to tags
+    extract_endorsements_to_tags: bool = pydantic.Field(
+        default=True, description="Whether to extract endorsements to tags"
```

Comment:
Please warn the author that enabling this will potentially overwrite Tags defined inside the DataHub application

Reply:
You can also mention that there's an AddTagTransformer that supports semantic addition of tags (e.g. PATCH instead of REPLACE)